modules/OutboundEmailAccounts/OutboundEmailAccounts.php

```diff
@@ -172,15 +182,31 @@ function sendTestEmail()
 					//Hide the email address window and show a message notifying the user that the test email is being sent.
 					EmailMan.testOutboundDialog.hide();
 					overlay("{$APP['LBL_EMAIL_PERFORMING_TASK']}", "{$APP['LBL_EMAIL_ONE_MOMENT']}", 'alert');
-
+                    
 					var callbackOutboundTest = {
 						success	: function(o) {
 							hideOverlay();
 							var responseObject = YAHOO.lang.JSON.parse(o.responseText);
 							if (responseObject.status)
 								overlay("{$APP['LBL_EMAIL_TEST_OUTBOUND_SETTINGS']}", "{$APP['LBL_EMAIL_TEST_NOTIFICATION_SENT']}", 'alert');
-							else
-								overlay("Send Test Email", responseObject.errorMessage, 'alert');
+       							else {
+                                
+                                                           var dialogBody = 
```

Comment:
note only: we should refactoring this php file directly contains html, js etc. In future we should outsource these parts to separated template and javascript files. Also as I noticed this piece of code duplicated ish to: https://github.com/salesagility/SuiteCRM/pull/5077/files#diff-40025349886b0dbc1c7c5e25389783efR350

Reply:
Yeah, I mentioned this in my PR notes: the previous architecture and code are not helping me, and I don't have time to redesign so I just went along with it. I agree this should be improved in the future to avoid repetition, and to better separate HTML from Javascript, etc.

Reply:
Yeah I agree, and it was my mention here also as I started with "note only" :)